### DISCUSSION TITLE
Need a program that can be directly upgraded on iso or CD without reinstalling

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#437] [Ideas]
When there is a new beta build version, you can directly upgrade and keep your own installed software, without having to reinstall the software all the time.
needs to be like a video.
UI
https://youtu.be/l1vW0cOAfro?t=1009
Process
https://youtu.be/l1vW0cOAfro?t=1814

**Comment by probonopd** [comment]
Please don't open new tickets for things that already have existing tickets.
There are multiple ones regarding this:
* https://github.com/helloSystem/ISO/issues/55
* https://github.com/helloSystem/Utilities/issues/33
* https://github.com/helloSystem/Utilities/issues/34
